app/assets/stylesheets/footer.scss

```diff
@@ -1,5 +1,5 @@
 .footer-empty-div {
-  height: 250px;
+  height: 100px;
```

Comment:
This height seems to be small for the desktop and does not look good. 
Use media query or available bootstrap classes for responsive design. So that based on size of screen the height can be different